pkg/handler/repositories_test.go

```diff
@@ -115,7 +115,7 @@ func (suite *ReposSuite) SetupTest() {
 }
 
 func (suite *ReposSuite) TearDownTest() {
-	//Rollback and reset db.DB
+	// Rollback and reset db.DB
 	db.DB.Rollback()
 	db.DB = suite.savedDB
 }
```

Comment:
should this function even be here? I think this is archaic.

Reply:
very unneeded, removed